dipy/reconst/csdeconv.py

```diff
@@ -18,10 +19,55 @@
                               sph_harm_lookup, lazy_index, SphHarmFit,
                               real_sym_sh_basis, sh_to_rh, forward_sdeconv_mat,
                               SphHarmModel)
+from dipy.reconst.utils import _roi_in_volume, _mask_from_roi
 
 from dipy.direction.peaks import peaks_from_model
 from dipy.core.geometry import vec2vec_rotmat
 
+from dipy.utils.deprecator import deprecate_with_version
+
+
+@deprecate_with_version("dipy.reconst.csdeconv.auto_response is deprecated, "
+                        "Please use "
+                        "dipy.reconst.csdeconv.auto_response_ssst instead",
+                        since='1.2', until='1.4')
+def auto_response(gtab, data, roi_center=None, roi_radii=10, fa_thr=0.7):
```

Comment:
- Why this function need to be renamed ?
- the previous signature is not respected, for backward compatibilty, all previous parameters need to be present
- we need to warn the users about this change of parameter name (`roi_radii` instead of `roi_radius`).  I do not know what will be the best way to do it. Do you have an opinion @arokem ? I propose to add `roi_radius` and if the default value change, just warn the users that the value will be overwrite by roi_radii and to use roi_radii instead.

Reply:
@skoudoro the name needs to be changed due to different `auto_response` functions for `SSST` and `MSMT` versions I guess...

Reply:
Exactly, I wanted to differentiate between `SSST` and `MSMT`. For `roi_radii`, I propose to change it to `roi_radius` in the deprecated version `auto_response`.

Reply:
ok, I'm fine, just make sure to respect the previous signature here and warn the user if there is some parameters not used anymore

Reply:
Is it ok how I put the warning in the `auto_response` function?

Reply:
Yes, Great! Thank you.